src/routes/api.js

```diff
@@ -1,7 +1,8 @@
 const express = require("express");
 const router = express.Router();
 const { businesses, businessesId, newReview } = require("../controllers/api");
-const { s3Controller } = require("../controllers/s3Controller");
+// const { s3Controller } = require("../controllers/s3Controller");
```

Comment:
wait why comment this out ?

Reply:
fixed

Reply:
Because without the keys it makes trouble.